praw/models/reddit/subreddit.py

```diff
@@ -1,20 +1,21 @@
 """Provide the Subreddit class."""
+import socket
+
 # pylint: disable=too-many-lines
```

Comment:
This comment should remain above the imports. Also, I'm not sure why there is a newline before the other system-level imports.

Reply:
I honestly don't know, but that's why automated tools should be reviewed, as they aren't perfect